content/examples/databricks-cluster-local.Rmd

```diff
@@ -0,0 +1,170 @@
+---
+title: "Option 2 - Working inside of Databricks"
+output:
+  html_document:
+    toc: true
+    toc_depth: 2
+    toc_float:
+      collapsed: false
+params:
+  width: 600
+---
+
+## Overview
+
+If the recommended path of connecting to Spark remotely with Databricks Connect
+does not apply to your use case, then you can install RStudio Server Pro
+directly within a Databricks cluster.
```

Comment:
```suggestion
directly within a Databricks cluster. See instructions for [AWS](https://docs.databricks.com/spark/latest/sparkr/rstudio.html) or [Azure](https://docs.microsoft.com/en-us/azure/databricks/spark/latest/sparkr/rstudio).
```

Reply:
Added these links to the "Additional information" section of this page in c9f784b. Thanks!